docs/Reference/API-Methods.md

```diff
@@ -743,7 +743,7 @@ None
 
 ### eth_gasPrice
 
-Returns the current gas unit price in wei.
+Returns the current gas unit price in a running node in wei. It is the hexadecimal equivalent of the price specified for the [`--min-gas-price`](CLI/CLI-Syntax.md#min-gas-price) command line option when the node was started or the default.
```

Comment:
Returns the current gas unit price in a running node in wei -> Returns the current gas unit price in wei

_running node_ doesn't make sense there because the node has to be running in order to make a JSON-RPC request. It's relevant in the min-gas-price description because you can't see min-gas-price in a running node. 

the default -> the default minimum gas price 

It's not clear what default this is currently

Reply:
Updated.